README.md

```diff
@@ -109,3 +109,24 @@ compile 'org.eclipse.collections:eclipse-collections-forkjoin:7.0.0'
 
 [license-edl]:LICENSE-EDL-1.0.txt
 [license-edl img]:https://img.shields.io/badge/License-EDL-blue.svg
+
+## License
+Eclipse Collections is open sourced under the Eclipse Public License v1.0 and the Eclipse Distribution License v1.0.
+
+## How to Contribute
+We welcome contributions to the Eclipse Collections project!!
+
+The Eclipse Collections project accepts contributions via pull requests in this GitHub. Please see [How To Contribute](CONTRIBUTING.md) and [Developers Guide](https://github.com/eclipse/eclipse-collections/wiki/DevelopersGuide) for more detail.
+
+## Project Roadmap
+* Project Roadmap: https://github.com/itohro/eclipse-collections/wiki/Roadmap
+
+## Links
+* Project Website: http://www.eclipse.org/collections
+* Eclipse PME: https://projects.eclipse.org/projects/technology.collections
+* GitHub: https://github.com/eclipse/eclipse-collections
+* Issues: https://github.com/eclipse/eclipse-collections/issues
+* Mailing lists: https://dev.eclipse.org/mailman/listinfo/collections-dev
+* Wiki: https://github.com/eclipse/eclipse-collections/wiki
+* Forum: https://www.eclipse.org/forums/index.php?t=thread&frm_id=329
+* StackOverflow: http://stackoverflow.com/questions/tagged/eclipse-collections
```

Comment:
This new content at the bottom of the README.md looks good. It should be moved up a few lines so that the link targets (for badges and stuff like that) are at the bottom of the file, like footnotes.